src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -129,19 +133,30 @@
                         </ul>
                     </div>
                 </li>
-                <li><span class="moreInfoPointTitle">Enrolling without spreadsheets</span>
+                <li>
+                    <span class="moreInfoPointTitle">Enrolling without spreadsheets</span>
                     <div class="moreInfoPointDetails">
                         The alternative is to type student data in the text box, using the pipe symbol (also called the vertical bar, 
                         not to be confused with upper case i or lower case L) to separate values.
                         <br> Here is an example.
-                        <br><br>
+                        <br>
+                        <br>
+                        
                         <div id="moreInfoEnrollWOSpreadSheetEg">
                             <samp>
-                            <span class="enrollLines"><b>Section   |   Team   |   Name   |   Email   |   Comments</b></span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 1   |   Tom Jacobs  |  tom@email.com</span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 1  |   Jean Wong   |   jean@email.com   |   Exchange Student</span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 2   |   Jack Wayne  |  jack@email.com</span>
-                            <br><span class="enrollLines">Tut Group 2   |   Team 3   |   Thora Parker  |  thora@email.com</span>
+                                <span class="enrollLines"><b>Section   |   Team   |   Name   |   Email   |   Comments</b></span>
+                                <br>
```

Comment:
It's more meaningful to put the `<br>` at the end of the same line because it means a line break.